src/Sai2Model.cpp

```diff
@@ -1033,6 +1033,44 @@ void Sai2Model::displayLinks() {
 	cout << endl;
 }
 
+void Sai2Model::addLoad(const std::string& link_name,
+						const double& mass,
+						const Vector3d& com_pos,
+						const Matrix3d& inertia,
+						const std::string& body_name) {
+	if (_load_names_to_load_mass_map.find(body_name) != _load_names_to_load_mass_map.end()) {
+		std::cout << "Load with name already added; skipping\n";
+		return;
+	} 
+	RigidBodyDynamics::Math::SpatialTransform joint_frame = \
+		RigidBodyDynamics::Math::SpatialTransform(Matrix3d::Identity(), Vector3d(0, 0, 0));
+	RigidBodyDynamics::Joint joint = RigidBodyDynamics::Joint(RigidBodyDynamics::JointTypeFixed);
+	RigidBodyDynamics::Body body = \
+		RigidBodyDynamics::Body(mass, RigidBodyDynamics::Math::Vector3d(com_pos), RigidBodyDynamics::Math::Matrix3d(inertia));
+	_rbdl_model->AddBody(linkIdRbdl(link_name), joint_frame, joint, body, body_name);
+	_load_names_to_load_mass_map.insert(std::make_pair(body_name, LinkMassParams(mass, com_pos, inertia, link_name)));
+}
+
+void Sai2Model::Sai2Model::removeLoad(const std::string body_name) {
```

Comment:
load_name instead of body_name to be consistent with the function name (removeLoad)